src/main/java/gov/nih/nci/evs/api/service/ElasticOperationsServiceImpl.java

```diff
@@ -139,4 +143,16 @@ public String delete(String indexName, String id) {
   public String deleteIndexMetadata(String id) {
     return operations.delete(id, IndexCoordinates.of(METADATA_INDEX));
   }
+
+  /** */
+  @Override
+  public void deleteQuery(String query, String indexName) {
```

Comment:
This is returning a void. How are we communicating that the query was deleted successfully?

Reply:
The function is contained within a try-catch which indicates if the delete fails. I'll add a boolean to return that can also indicate success/failure.